packages/plugin-commands-audit/src/fix.ts

```diff
@@ -4,7 +4,7 @@ import fromPairs from 'ramda/src/fromPairs'
 
 export async function fix (dir: string, auditReport: AuditReport) {
   const { manifest, writeProjectManifest } = await readProjectManifest(dir)
-  const vulnOverrides = createOverrides(Object.values(auditReport.advisories))
+  const vulnOverrides = createOverrides(Object.values(auditReport.advisories), manifest.pnpm?.allowList)
```

Comment:
Put the new setting into `auditConfig`. And maybe a better name would be ignoreCves
```suggestion
  const vulnOverrides = createOverrides(Object.values(auditReport.advisories), manifest.pnpm?.auditConfig?.ignoreCves)
```